sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -131,7 +205,72 @@ export interface ExtractKeyPhrasesSuccessResult extends TextAnalyticsSuccessResu
 }
 
 // @public
-export type InnerErrorCodeValue = "InvalidParameterValue" | "InvalidRequestBodyFormat" | "EmptyRequest" | "MissingInputRecords" | "InvalidDocument" | "ModelVersionIncorrect" | "InvalidDocumentBatch" | "UnsupportedLanguageCode" | "InvalidCountryHint";
+export interface HealthcareEntitiesArray extends Array<HealthcareResult> {
+}
+
+// @public (undocumented)
+export type HealthcareEntity = Entity & {
+    isNegated: boolean;
+    links?: HealthcareEntityLink[];
+};
+
+// @public (undocumented)
+export interface HealthcareEntityLink {
+    dataSource: string;
+    id: string;
+}
+
+// @public
+export type HealthcareErrorResult = TextAnalyticsErrorResult;
+
+// @public
+export interface HealthcareJobOptions extends TextAnalyticsOperationOptions {
+}
+
+// @public (undocumented)
+export interface HealthcareRelation {
+    bidirectional: boolean;
+    relationType: string;
+    source: string;
+    target: string;
+}
+
+// @public
+export type HealthcareResult = HealthcareSuccessResult | HealthcareErrorResult;
+
+// @public
+export interface HealthcareSuccessResult extends TextAnalyticsSuccessResult {
+    entities: HealthcareEntity[];
+    relations: HealthcareRelation[];
+}
+
+// @public
+export type HealthPollerLike = PollerLike<BeginAnalyzeHealthcareOperationState, PaginatedHealthcareEntities>;
+
+// @public
+export type InnerErrorCodeValue = "InvalidParameterValue" | "InvalidRequestBodyFormat" | "EmptyRequest" | "MissingInputRecords" | "InvalidDocument" | "ModelVersionIncorrect" | "InvalidDocumentBatch" | "UnsupportedLanguageCode" | "InvalidCountryHint" | string;
```

Comment:
not familiar with JS but want to make sure that the addition of `string` at the end was intentional

Reply:
yeah, this is part of the generated code according to latest swagger.